src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1983,4 +1983,4 @@ Additional information: {1}</value>
   <data name="Stack_Trace" xml:space="preserve">
     <value>Stack Trace</value>
   </data>
-</root>
\ No newline at end of file
+</root>
```

Comment:
Hooray for editing in the browser. I can undo this if you want or we can leave it to make updating strings from GitHub easier.